modules/custom/falcon_donation/src/EventSubscriber/FalconDonationOrderCompletedSubscriber.php

```diff
@@ -76,10 +81,14 @@ public function orderCompleteHandler(WorkflowTransitionEvent $event) {
 
       // Check subject and body for thank you email.
       if (empty($subject) || empty($body)) {
+        $this->logger
+          ->info(
+            'Can`t send Thank you email because thankyou_email_subject or thankyou_email_body is empty. Appeal id = @appeal_id. Order id = @order_id',
```

Comment:
1. It's an error, not info.

2. The text should be `Could not send Thank You email, because the Thank You email subject or body is empty. Related appeal ID is @id, related order ID is @id`